metals/src/main/resources/millw.bat

```diff
@@ -80,4 +81,15 @@ if not exist "%MILL%" (
 set MILL_DOWNLOAD_PATH=
 set MILL_VERSION=
 
-"%MILL%" %*
+set MILL_PARAMS=%*
+
+if defined STRIP_VERSION_PARAMS (
+    for /f "tokens=1-2*" %%a in ("%*") do (
```

Comment:
So what happens here in this somewhat cryptic code:

1) `for /f` - is used for processing text by breaking the line up into individual items of data called 'tokens'. By default breaks occur at a space character.
2) `"tokens=1-2*"` - this specifies that the first 2 parameters in our text should be allocated to the first 2 letters, i.e "%%a" and "%%b" and the asterisk specifies that the remaining parameters should be allocated to the letter "%%c".
3) `%%a` - a replaceable parameter. We start at "%%a" for the first parameter, then "%%b" for the second and so on.
4) `in ("%*")` - text string that we are processing